README.md

```diff
@@ -148,7 +148,7 @@ subnet-cli wizard \
 --chain-name=subnetevm
 ```
 
-_The `vm-id` was generated by calling `subnet-cli create VMID wagmi`. You can
+_The `vm-id` was generated by calling `subnet-cli create VMID subnetevm`. You can
 use any value here, the only important thing is to make sure the binary you
 generate has the same name._
```

Comment:
nit: should we remove the underscore after the period here?

Reply:
I think it's required for _italic_.